weldx/tags/core/data_array.py

```diff
@@ -12,8 +12,7 @@
 class XarrayDataArrayConverter(WeldxConverter):
     """Serialization class for xarray.DataArray."""
 
-    name = "core/data_array"
-    version = "0.1.0"
+    tags = ["asdf://weldx.bam.de/weldx/tags/core/data_array-0.1.*"]
```

Comment:
should these now be tuples as well (as the type hint in the base class suggests.). I added this in #727, since I thought it make more sense to indicate that this is immutable.

Reply:
true, probably `tuple` is fitting

I only used `list` here because this is how it is in the asdf codebase.
I **think** the `tags` attribute gets 'resolved' in asdf and ends up as a list anyway. I don't know if that breaks with tuples here.

Frankly I would just stick to what asdf is doing to avoid any troubles if we can